packages/renderer/src/lib/image/BuildImageFromContainerfile.svelte

```diff
@@ -113,6 +113,13 @@ async function getContainerfileLocation() {
   if (!result.canceled && result.filePaths.length === 1) {
     containerFilePath = result.filePaths[0];
     hasInvalidFields = false;
+
+    // Extract the name of the folder containing the Containerfile
+    const folderName = containerFilePath.split('/').slice(-2, -1)[0];
```

Comment:
Hello I'm not sure it may work on Windows as paths may have backslashes

maybe should you use path.basename and path.dir ale methods containerFilePath) instead

Reply:
Ah whoops rookie mistake, I'm not too familiar with svelte however, what's the best way to import path here?

Reply:
Ah I'm not sure if possible because Svelte component can't access Node API

Reply:
ah yes correct, you'll need to use custom function like you did but it requires to work with Windows UNC style file I guess